### ISSUE TITLE
Diamond Flask needs to trigger GCD

### ISSUE THREAD

**Issue by Jinnai** [timhul/ClassicSim#203]
Diamond Flask triggers GCD which does not seem to be simulated by ClassicSim.

Maybe this is an issue for other trinkets as well, as there are some trinkets that trigger GCD and some that don't.

**Comment by timhul** [comment]
You are correct that the tool does not simulate it. In order to make this scalable the trinket item files themselves should support a GCD attribute/element to propagate this information.
